src/notation/view/widgets/editstyle.ui

```diff
@@ -5874,74 +5917,38 @@
-                    <property name="keyboardTracking">
-                     <bool>false</bool>
-                    </property>
-                    <property name="suffix">
-                     <string>sp</string>
-                    </property>
-                    <property name="minimum">
-                     <double>2.000000000000000</double>
-                    </property>
-                    <property name="value">
-                     <double>4.000000000000000</double>
-                    </property>
-                   </widget>
-                  </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Cap width after:</string>
                     </property>
                    </widget>
                   </item>
-                  <item row="2" column="1">
-                   <widget class="QSpinBox" name="mmRestRefDuration">
-                    <property name="suffix">
-                     <string> / 4</string>
-                    </property>
-                    <property name="minimum">
-                     <number>2</number>
-                    </property>
-                    <property name="maximum">
-                     <number>8</number>
+                  <item row="2" column="3">
+                   <widget class="QToolButton" name="resetMMRestRefDuration">
+                    <property name="text">
+                     <string notr="true"/>
                     </property>
                    </widget>
                   </item>
-                  <item row="4" column="2">
-                   <widget class="QToolButton" name="mmRestMaxMeasuresReset">
+                  <item row="2" column="0">
+                   <widget class="QRadioButton" name="mmRestWidthConstant">
                     <property name="text">
-                     <string notr="true"/>
+                     <string>Fixed, based on time signature of:</string>
                     </property>
                    </widget>
                   </item>
-                  <item row="4" column="1">
-                   <widget class="QSpinBox" name="mmRestMaxMeasures">
-                    <property name="suffix">
-                     <string> measures</string>
+                  <item row="4" column="2">
+                   <widget class="QLabel" name="label_2">
+                    <property name="text">
+                     <string>measure(s)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="2">
-                   <widget class="QToolButton" name="resetMMRestRefDuration">
+                   <widget class="QLabel" name="label_17">
                     <property name="text">
-                     <string notr="true"/>
+                     <string>quarters</string>
```

Comment:
I think `/ 4` would be clearer here. Perhaps we should even keep it inside the spinbox and make it untranslatable.

Reply:
We have numbers being translatable elsewhere. "Arabic" numbers are not used all over the world  and esp. not in Arab speaking countries...
But `/ 4` is of course possible.

Reply:
Inside the spinbox is having the issue of needing the leading space. Could be `/4` though, no spaces at all

Reply:
Hmm, yes, that seems right

Reply:
done